src/common/cluster-store.ts

```diff
@@ -43,8 +44,10 @@ export interface ClusterModel {
   ownerRef?: string;
   accessibleNamespaces?: string[];
 
-  /** @deprecated */
-  kubeConfig?: string; // yaml
+  /**
+   * @deprecated
+   */
+  kubeConfig?: string;
```

Comment:
Not related to this PR but should we actually remove this now that we are doing major version bump.

Reply:
I tried. But it messes with the migrations and I didn't want to try and figure it out. I think we should try to at least.